packages/webpack-model/src/index.ts

```diff
@@ -32,11 +32,11 @@ export function prepareWithJora(
     ...options.helpers,
     ...joraHelpers(compilations),
   });
-  const context = {};
 
   return {
     files,
     compilations: compilations.map((c) => c.data),
-    query: (query: string, data: unknown): unknown => j(query)(data || files, context),
+    query: (query: string, data: unknown, context: unknown = {}): unknown =>
```

Comment:
I thank that creating a new object every time that we calling `query` without a context - not a good idea )
```suggestion
    query: (query: string, data: unknown, context: unknown = rootContext): unknown =>
```